DateSeer/Matches.cs

```diff
@@ -93,13 +102,27 @@ private void UploadUser(User person, int brake1)
                 label6.Text = person.getemail();
```

Comment:
Using methods that were changed in user class by me. Will update and resolve merge conflicts regarding this.